External/FEXCore/Source/Interface/Core/JIT/x86_64/JITClass.h

```diff
@@ -152,6 +152,8 @@ class X86JITCore final : public CPUBackend, public Xbyak::CodeGenerator {
 
   static uint64_t ExitFunctionLink(X86JITCore* code, FEXCore::Core::CpuStateFrame *Frame, uint64_t *record);
 
+  void EmitDetectionString();
```

Comment:
Do you mind adding a comment in the code as well as to why this function exists? It looks a bit out-of-place without context, and it's hard to see that it's merely a debugging aid (?) rather than something that's being relied on upon elsewhere.